helm-charts/metal3-deploy/values.yaml

```diff
@@ -158,3 +158,8 @@ metal3-ironic:
 
     # If no dhpHosts set, all mac addresses will be acknowledged
     dhcpHosts: ""
+
+    # Base URI from where to pull the IPA images.
+    # Customer needs to host ironic-python-agent.tar and
```

Comment:
Suggest move the word `customer`. Maybe something like this?

The base URL to download IPA images (i.e. ironic-python-agent.tar and ironic-python-agent.tar.md5).